doc/manual/core_device.rst

```diff
@@ -12,84 +12,38 @@ Flash storage
-|                       | assumed.                                                  |
-+-----------------------+-----------------------------------------------------------+
-| ``device_map``        | If set, allows the core log to connect RTIO channels to   |
-|                       |                                                           |
-|                       | device names and use device names as well as channel      |
-|                       |                                                           | 
-|                       | numbers in log output. A correctly formatted table can be |
-|                       |                                                           |
-|                       | automatically generated with ``artiq_rtiomap``, see       |
-|                       |                                                           |
-|                       | :ref:`Utilities<rtiomap-tool>`.                           |
-+-----------------------+-----------------------------------------------------------+
-| ``net_trace``         | If set to ``1``, will activate net trace (print all       |
-|                       |                                                           | 
-|                       | packets sent and received to UART and core log). This     |
-|                       |                                                           |
-|                       | will considerably slow down all network response from the |
-|                       |                                                           |
-|                       | core. Not applicable for ARTIQ-Zynq (Kasli-SoC, ZC705).   |
-+-----------------------+-----------------------------------------------------------+
-| ``panic_reset``       | If set to ``1``, core device will restart automatically   |
-|                       |                                                           |
-|                       | upon panic. Not applicable for ARTIQ-Zynq.                |
-+-----------------------+-----------------------------------------------------------+
-| ``no_flash_boot``     | If set to ``1``, will disable flash boot. Network boot is |
-|                       |                                                           |
-|                       | attempted if possible. Not applicable for ARTIQ-Zynq.     |
-+-----------------------+-----------------------------------------------------------+
-| ``boot``              | Allows full firmware/gateware (``boot.bin``) to be written|
-|                       |                                                           |
-|                       | with ``artiq_coremgmt``, on ARTIQ-Zynq systems only.      |
-+-----------------------+-----------------------------------------------------------+
-
+``idle_kernel``
+  Stores (compiled ``tar`` or ``.elf`` binary of) idle kernel. See :ref:`miscellaneous_config_core_device`. 
```

Comment:
nitpick with the dot: either ``tar`` or ``elf`` or ``.tar`` or ``.elf``

Reply:
Eek.